package.json

```diff
@@ -102,6 +102,7 @@
     "po-catalog-loader": "2.0.0",
     "prism-sentry": "^1.0.2",
     "prop-types": "^15.6.0",
+    "qrcode.react": "^1.0.1",
```

Comment:
Can we replace https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/qrcode.tsx with this?

Reply:
@EvanPurkhiser I was considering that component but it looked like it only supports some sort of matrix data as a prop from authenticator - does it work with regular strings?